js/button.js

```diff
@@ -1,96 +1,93 @@
-export default class Button extends PIXI.Container {
-
+export default class Button extends PIXI.Sprite {
     constructor(options = {}) {
         super();
-        this.options = this.setOptions(options);
-        this.init();
+        this._options = options;
+        this._init();
     }
 
-    setOptions(options) {
-        const defaultOptions = {
-            width: 160,
-            height: 40,
-            texture: null,
-            x: 0,
-            y: 0,
-            fill: 0xFF3300,
-            label: 'Default button',
-            labelStyle: {
-                fill: '#ffffff',
-                fontSize: '16px',
-                fontFamily: 'Arial, sans-serif',
-            },
-            disabled: false,
-        };
-
-        return {...defaultOptions, ...options};
+    _init() {
+        this.buttonMode = true;
+        this._isActive = false;
```

Comment:
According to the way it's used, this is "_isPressed" rather than "_isActive" flag.